src/response/chi0.jl

```diff
@@ -42,16 +42,16 @@ function compute_χ0(ham; temperature=ham.basis.model.temperature)
     basis = ham.basis
     filled_occ = filled_occupation(basis.model)
     smearing = basis.model.smearing
-    n_spin   = basis.model.n_spin_components
```

Comment:
You have a lot of these whitespace commits in your pr. Can you make sure you keep the original formatting where appropriate ?

Reply:
Fixed

Reply:
Thanks :+1: